admin-ui/plugins/auth-server/helper/validations.js

```diff
@@ -0,0 +1,25 @@
+// LDAP validation schema using Yup
+import * as Yup from 'yup'
+import { STRINGS } from './constants'
+
+export function getLdapValidationSchema() {
+  const e = STRINGS.authn.ldap.errors
+  return Yup.object().shape({
+    configId: Yup.string().min(1, e.acr).required(e.acr),
+    bindDN: Yup.string().min(1, e.bind_dn).required(e.bind_dn),
+    bindPassword: Yup.string().min(1, e.bind_password).required(e.bind_password),
+    servers: Yup.array()
+      .of(Yup.string().min(1, e.remote_ldap_server).required(e.remote_ldap_server))
+      .min(1, e.remote_ldap_server)
+      .required(e.remote_ldap_server),
+    baseDNs: Yup.array()
+      .of(Yup.string().min(1, e.base_dns).required(e.base_dns))
+      .min(1, e.base_dns)
+      .required(e.base_dns),
+    maxConnections: Yup.number().typeError(e.max_connections_type).required(e.max_connections),
```

Comment:
Add positive number validation for maxConnections

The maxConnections field should validate that the number is positive to prevent invalid configuration.



```diff
-    maxConnections: Yup.number().typeError(e.max_connections_type).required(e.max_connections),
+    maxConnections: Yup.number()
+      .typeError(e.max_connections_type)
+      .positive('Max connections must be a positive number')
+      .integer('Max connections must be an integer')
+      .required(e.max_connections),
```